### DISCUSSION TITLE
How to deploy to Vercel with one click?

### DISCUSSION THREAD

**Discussion by dishengai** [mckaywrigley/chatbot-ui#1299] [Ideas]
How to deploy to Vercel with one click?

**Comment by yashsinha-aero** [comment]
U cant. You would need to set up a supabase database since local storage is no longer used.

**Reply by dishengai** [reply]
oo 3q

**Comment by Itangalo** [comment]
You probably could use the old code, on the Legacy branch: https://github.com/mckaywrigley/chatbot-ui/tree/legacy

The 2.0 version includes more functionality, but also uses Supabase for database. There is an instruction for "hosted quickstart" in the Readme on the project front page, but it requires much more work than the old version.

**Comment by masonjames** [comment]
This could be "nearly" replicated though... In practice, the software works on Vercel by default. Supabase also has a 1-click integration with Vercel as well. 

So it's something like:
1. Create projects on both vercel and supabase 
2. Use the migration scripts in chatbot-ui repo to configure your supabase project
3. Setup the vercel integration from within your supabase project
4. Redeploy on vercel
